### PR TITLE
Fix : minor bugs

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

File: get-started.html [hunk 1]
```diff
@@ -100,13 +100,13 @@
...
             </li>
             <li>
               <button><a href="./docs/List.html">List</a></button>
             </li>
             <li>
-              <button><a href="./docs/Modal.html">Moadal</a></button>
+              <button><a href="./docs/Modal.html">Modal</a></button>
             </li>
             <li>
               <button><a href="./docs/navbaar.html">Nabar</a></button>
```
Comment (hinanshisuthar) [inline get-started.html:113]: You mean Navbar here?

**Review by hinanshisuthar — COMMENTED** [review]
Looks good to me. Ready to merge.

**Review by Ronak-Mutha — COMMENTED** [review]
Code looks clean and good use of variables. Good to merge

File: get-started.html (hunk 1 above)
Comment (ZuberDunge) [reply get-started.html:113]: yes. updated.

**Pull request merged**